packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -115,10 +123,10 @@ function findNativeHandler({ domTreeShapes, start, current, anchor }) {
       shape[axisProperties.scrollLength[axis]];
 
     if ((goingForward && areNotAtEnd) || (!goingForward && areNotAtStart)) {
-      return shape;
+      return true;
     }
 
-    return null;
+    return false;
```

Comment:
These are return values of `some(() => shape)` so we might as well directly cast them.